src/components/modules/chat/hooks/use-all-chats.hook.ts

```
@@ -0,0 +1,75 @@
+          unsubscribe();
+        };
+      } catch (err) {
+        console.error("Error setting up chats:", err);
+        setError("Error setting up chats");
+        setLoading(false);
+      }
+    };
+
+    const cleanup = setupChats();
+    return () => {
+      cleanup.then((unsubscribe) => unsubscribe && unsubscribe());
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)